### DISCUSSION TITLE
Perguntas

### DISCUSSION THREAD

**Discussion by fortecria** [fortecria/Paraachiev#1] [Q&A] [answered]
Façam perguntas

**Comment by henzofelici** [comment, marked as answer]
Com toda certeza amigão

**Comment by fortecria** [comment]
Como respondo aqui?

**Reply by henzofelici** [reply]
apenas escreva no campo em branco e clique em reply

**Reply by fortecria** [reply]
deu certo muito obrigado

**Comment by henzofelici** [comment]
apenas escreva no campo em branco e clique em reply
